apps/macos/src/Podfile

```diff
@@ -27,6 +27,9 @@ fi
 use_flipper!(false)
 use_test_app! do |target|
   target.app do
+
+    pod 'RNCPicker', :path => '../../../node_modules/@react-native-community/picker/RNCPicker.podspec'
```

Comment:
Same as comment in the iOS Podfile. We shouldn't need to do this and we should figure out why autolinking isn't working.